30 Days of Code (Python)/D25-Running-Time-and-Complexity.py

```diff
@@ -69,15 +69,15 @@
     n = int(input())
     for i in range(n):
         nums.append(int(input()))
-for _ in nums:
-    if _ == 1 or _ == 0:
-        print('Not prime')
-    else:
-        m=0
-        for a in range(2,int(math.sqrt(_))+1):
-            if _%a == 0:
-                m+=1
-        if m == 0:
-            print('Prime')
+    for _ in nums:
```

Comment:
Can we please change `_` to `num`?  The single underscore character is reserved in Python for variables that we will not reuse but this variable is referred to on lines 73, 77, and 78.

Reply:
That would be great.